backend/resources/hacknight.py

```diff
@@ -54,3 +54,38 @@ def get(self, id):
         return {'hacknights': hacknight_schema.dump(Hacknight.query.get_or_404(
             id))
         }, HTTPStatus.OK
+
+    @jwt_required
+    def patch(self, id):
```

Comment:
didn't we agree to have separate url for that (like `/hacknights/<id>/participants`)?

Reply:
I hear it for first time. No one discussed it with me, although there was plenty of time - it is 4 months since I proposed this pr.

Reply:
changed